lib/eldap/src/eldap.erl

```diff
@@ -415,6 +478,12 @@ Modify an entry.
 ```
 """.
 -doc(#{since => <<"OTP R15B01">>}).
+-spec modify(Handle, Dn, ModifyOps) -> ok | {ok, Value} | {error, Reason} when
+    Handle :: handle(),
+    Dn :: string(),
+    ModifyOps :: [term()],
+    Value :: term(),
```

Comment:
According to the old doc this function can also return "{ok, {referral, referrals()}}",
which 'Value" must reflect.

Reply:
fixed